resources/js/home.js

```diff
@@ -0,0 +1,41 @@
+(function(slides, dots, prevElem, nextElem) {
```

Comment:
Create separate JS for carousel and include it in JS for home page. This will allow you to use carousel on any page.

Reply:
done